CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Mapbox welcomes participation and contributions from everyone.
 ## Unreleased
 #### Features
 #### Bug fixes and improvements
+- Started cancelling reroute request when the user returns to the route. [#6614](https://github.com/mapbox/mapbox-navigation-android/pull/6614)
```

Comment:
```suggestion
- Fixed rerouting: interrupt reroute request if the puck returns to the route. [#6614](https://github.com/mapbox/mapbox-navigation-android/pull/6614)
```

Reply:
IMO it's not a fix, it's an optimization.

Reply:
`Changed rerouting: ...`?

Reply:
Changed to "optimized".